### ISSUE TITLE
Drop support for sqlplus, sqlcl Liquibase ci/cd

### ISSUE THREAD

**Issue by yevon** [mickeypearce/oradew-vscode#63]
What do you think about dropping support for sqlplus? I say it because I plan to add sqlcl liquibase ci/cd compatibility in liquibase, and for me I don't see any advantages of mantaining two different implementations as sqlcl can do everything and more than sqlplus can do.

**Comment by true-hb** [comment]
sorry, I need sqlplus, because we are still working with Oracle 11 in our company. I guess, I'm not the only one.

**Comment by yevon** [comment]
mmm ok, we will have to limit some features only if you have installed sqlcl.

**Comment by mickeypearce** [comment]
Yes, no problem, I think that SqlCl is set to execute by default, so no problem if some features don't work if you switch to sqlplus.

**Comment by yevon** [comment]
Wow very interesting, I was waiting for this as standard Liquibase has support for this. I will try to have a look at it. We might even replace curent schema sql generator with sqlcl generator. I found it quite slow generating changelogs, it might have improved. I will have a look at this.

**Comment by yevon** [comment]
I will just create two new command, like import liquibase and deploy liquibase for keeping current compatibility.
